wpilibc/src/main/native/include/frc/DoubleSolenoid.h

```diff
@@ -88,6 +88,39 @@ class DoubleSolenoid : public wpi::Sendable,
    */
   void Toggle();
 
+  /**
+   * Returns true if the double solenoid is in a forward state.
+   * @return true if the double solenoid is in a forward state.
+   */
+  virtual bool IsForward() const;
```

Comment:
These don't need to be virtual

Reply:
I noticed that as well.  Why are they virtual for Solenoid's Get/Set

Reply:
Relic from SolenoidBase. Just opened #7080

Reply:
Also should get/set be on the way to deprecation because of these new methods?